pol-core/pol/mobile/charactr.cpp

```diff
@@ -3177,6 +3177,9 @@ void Character::set_warmode( bool i_warmode )
             return;
           if ( !chr->in_visual_range( this ) )
             return;
+          if ( !chr->is_visible_to_me( this ) )
```

Comment:
`is_visible_to_me` by default checks also `in_visual_range` so the check before can be removed.

Reply:
thanks, fixed